dashboards/dashboard_2/README.md

```diff
@@ -0,0 +1,72 @@
+# Streamlit Dashboard for Open Source Quartz Solar Forecast
+
+This Streamlit Dashboard provides an interactive interface for running and visualizing solar forecasts using the [Open Source Quartz Solar Forecast](https://github.com/openclimatefix/Open-Source-Quartz-Solar-Forecast) model developed by [Open Climate Fix](https://openclimatefix.org/).
+
+
+https://github.com/user-attachments/assets/940ca4a7-65a4-40ea-a602-ea0f65e96746
+
+
+## Features
+
+- Configure PV site parameters (latitude, longitude, capacity)
+- Select inverter type (No Inverter or Enphase)
+- Enphase API authentication flow (if applicable)
+- Run solar forecast
+- Visualize forecast results with interactive charts
+- Compare forecasts with and without recent PV data (if applicable)
+- Display raw forecast data and provide an option to download it as CSV
+
+## How to Run
+
+1. Clone the repository and install all the dependencies in a virtual environment on a Linux System(or WSL):
+   `pip install -e .` and `pip install -r requirements.txt`
+
+2. Set up environment variables (if applicable):
+
+- Create a `.env` file in your root directory
+- Add the following variables:
+  ```
+  ENPHASE_CLIENT_ID=your_client_id
+  ENPHASE_CLIENT_SECRET=your_client_secret
+  ENPHASE_API_KEY=your_api_key
+  ENPHASE_SYSTEM_ID=your_system_id
+  ```
+
+3. Navigate to the `dashboards/dashboard_2` directory.
+
+4. Run the Streamlit app: `streamlit run app.py`
```

Comment:
Would it also not be possible to run the dashboard from just `pip install pip install quartz-solar-forecast` and then running something like `streamlit run quartz_solar_forecast.dashboards.dashboard_1.app.py` ?

Reply:
> Would it also not be possible to run the dashboard from just `pip install pip install quartz-solar-forecast` and then running something like `streamlit run quartz_solar_forecast.dashboards.dashboard_1.app.py` ?

it is possible, but for any user who's never coded as much, it makes sense to just go to the directory and run it so they know what theyre running

Reply:
I think for someone who hasn't coded much, just running a pip install of library, then one command i.e `streamlit run quartz_solar_forecast.dashboard.py` would much easier then cloning the repo and the few extra steps it would take. Maybe we can look at this again in the future!